src/gitgraph.js

```diff
@@ -1205,6 +1207,19 @@
       }
     }
 
+    if (this.messageShort !== undefined) {
```

Comment:
You can write `if (this.messageShort)` here, to be consistant with the actual code style.